sympy/physics/mechanics/actuator.py

```diff
@@ -990,3 +991,147 @@ def __repr__(self):
         return (f"{self.__class__.__name__}("
                 f"{self.linear_stiffness}, {self.nonlinear_stiffness}, {self.pathway}, "
                 f"equilibrium_length={self.equilibrium_length})")
+
+class CoulombKineticFriction(ForceActuator):
+    """Coulomb kinetic friction with Stribeck and viscous effect.
+
+    Explanation
+    ===========
+
+    This represents a Coulomb kinetic friction with the Stribeck and viscous effect,
+    described by the function:
+
+        F = f_c * sign(v) + (f_max - f_c) * exp(-(v / v_s)**2) + sigma * v
+
+    where `f_c` is the Coulomb friction constant, `f_max` is the maximum static friction force,
```

Comment:
```suggestion
    where `f_c` is the Coulomb friction constant, ``f_max`` is the maximum static friction force,
```

Reply:
I used :math:`` instead and checked the docstring locally. It looks fine to me.
<img width="693" alt="Screenshot 2024-07-18 at 02 57 01" src="https://github.com/user-attachments/assets/97a2e4ae-afc6-424a-ae32-f04b32751b96">
<img width="693" alt="Screenshot 2024-07-18 at 02 57 10" src="https://github.com/user-attachments/assets/5b05370d-d06c-4a01-a529-085458a6e7b0">
<img width="693" alt="Screenshot 2024-07-18 at 02 57 22" src="https://github.com/user-attachments/assets/4cb9fe2d-fb03-49e2-a527-4f95b770b2a0">